### ISSUE TITLE
Accessibility Issue with NVDA on chrome

### ISSUE THREAD

**Issue by Hamzayy** [ericgio/react-bootstrap-typeahead#443]
### Version

4.0.0-alpha.9

### Steps to reproduce

Accessibility issue on chrome with NVDA

### Expected Behavior

When we select an item from the list then voice reader (NVDA) should read that selected item before saying collapsed.

### Actual Behavior

When we select an item from the list then voice reader (NVDA) doesn't read that selected item and it only says collapsed. 

I've done enough R&D to get this done but all in vain. Please suggest me any solution.

Thanks

**Comment by ericgio** [comment]
Hi @Hamzayy, thanks for filing this issue. I don't see any recommendations in [the authoring practices](https://www.w3.org/TR/wai-aria-practices-1.1/#combobox) for selections, but please point me in the right direction if I'm wrong.

If you need an announcer, you can add your own by using a child render function. The function will receive the typeahead state, including selections, which you can use to set the announcement message.

> I've done enough R&D to get this done but all in vain.

Sorry, but not sure what that means. Were you able to get something working?

**Comment by ericgio** [comment]
Closing, since this isn't a bug and there's been no response from OP in >2 weeks.
